line-bot-model/src/main/java/com/linecorp/bot/model/event/Event.java

```diff
@@ -52,4 +53,19 @@
      * Time of the event
      */
     Instant getTimestamp();
+
+    /**
+     * Abstract interface for reply support events.
+     *
+     * @see Event
+     */
+    interface ReplySupport extends Event {
```

Comment:
Extract as `public interface Replyable`. (not extends)

There are many name candidates.

* `ReplySupport`
* `ReplySupportEvent`
* `Replayable`

I choose `Replayable`. feedback please~.